sdks/python/apache_beam/typehints/typehints.py

```diff
@@ -601,6 +604,13 @@ def __getitem__(self, py_type):
     return Union[py_type, type(None)]
 
 
+def is_optional(typehint):
```

Comment:
May be call this "is_nullable" to  be more specific ?

Reply:
good idea